<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>video_config</class>
 <widget class="QWidget" name="video_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_preset"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Camara">
      <attribute name="title">
       <string>Camara</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="grid_datas" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Brightness</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="Slider_contrast">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSlider" name="Slider_blackligth">
            <property name="maximum">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="comboBox_powerline">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>60Hz</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Sharpness</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSlider" name="Slider_whitebalance">
            <property name="minimum">
             <number>2800</number>
            </property>
            <property name="maximum">
             <number>6500</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>4600</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="Slider_brightness">
            <property name="minimum">
             <number>-64</number>
            </property>
            <property name="maximum">
             <number>64</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="Slider_gamma">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>300</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QCheckBox" name="checkBox_whitebalance">
            <property name="text">
             <string>White Balance, Automatic </string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="Slider_sharpness">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="Slider_hue">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Blackligth Compensation</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>White Balance Temperature</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Hue</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="Slider_saturation">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Contrast</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Power Line Frecuency</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Saturation</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="checkBox_exposure">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Exposure, dynamic framerate</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lbl_brightness">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="lbl_contrast">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="lbl_hue">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="lbl_gamma">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="lbl_saturation">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="lbl_sharpness">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="lbl_whitebalance">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="lbl_blackligth">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>HoughCircles</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>minDist</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>param1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>minRadius</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="cb_ksize_equals">
         <property name="text">
          <string>=</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="8">
        <widget class="QDoubleSpinBox" name="sb_param1_hough">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="8">
        <widget class="QSpinBox" name="sb_maxrad_hough"/>
       </item>
       <item row="2" column="5">
        <widget class="QSpinBox" name="sb_ksizew_hough"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>KsizeH</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Sigma</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>KsizeW</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Param2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="sb_param2_hough">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>30.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QDoubleSpinBox" name="sb_mindist_hough">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="sb_dp_hough">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="8">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Circles</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="sb_ksizeh_hough">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>dp</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QSpinBox" name="sb_minrad_hough"/>
       </item>
       <item row="2" column="7">
        <widget class="QDoubleSpinBox" name="sb_sigma_hough">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="7">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>MaxRadius</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="8">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Blurred</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Canny">
      <attribute name="title">
       <string>Canny</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Blobs</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Página</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_resetpreset">
     <property name="text">
      <string>Reset Preset</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_savepreset">
       <property name="text">
        <string>Save preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_newpreset">
       <property name="text">
        <string>Create new preset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_preset</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>sb_ksizeh_hough</tabstop>
  <tabstop>cb_ksize_equals</tabstop>
  <tabstop>sb_ksizew_hough</tabstop>
  <tabstop>sb_sigma_hough</tabstop>
  <tabstop>sb_dp_hough</tabstop>
  <tabstop>sb_param2_hough</tabstop>
  <tabstop>sb_mindist_hough</tabstop>
  <tabstop>sb_minrad_hough</tabstop>
  <tabstop>sb_param1_hough</tabstop>
  <tabstop>sb_maxrad_hough</tabstop>
  <tabstop>pushButton_resetpreset</tabstop>
  <tabstop>pushButton_savepreset</tabstop>
  <tabstop>pushButton_newpreset</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>Slider_whitebalance</tabstop>
  <tabstop>Slider_blackligth</tabstop>
  <tabstop>Slider_contrast</tabstop>
  <tabstop>Slider_hue</tabstop>
  <tabstop>checkBox_exposure</tabstop>
  <tabstop>checkBox_whitebalance</tabstop>
  <tabstop>comboBox_powerline</tabstop>
  <tabstop>Slider_brightness</tabstop>
  <tabstop>Slider_sharpness</tabstop>
  <tabstop>Slider_saturation</tabstop>
  <tabstop>Slider_gamma</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
